Sources/SimilaritySearchKit/Core/Index/SimilarityIndex.swift

```diff
@@ -17,9 +17,22 @@ public typealias TextSplitterType = SimilarityIndex.TextSplitterType
 public typealias VectorStoreType = SimilarityIndex.VectorStoreType
 
 @available(macOS 11.0, iOS 15.0, *)
-public class SimilarityIndex {
+public class SimilarityIndex: Identifiable, Hashable {
+	
+	public static func == (lhs: SimilarityIndex, rhs: SimilarityIndex) -> Bool {
+		return lhs.id == rhs.id
+	}
+	
+	
+	public func hash(into hasher: inout Hasher) {
+		hasher.combine(id)
+	}
+	
```

Comment:
What is the use case for this? Just curious.

Reply:
These methods provide conformance to Hashable, useful for SwiftUI integration. "ForEach" requires conformance to Hashable.